fastlane/actions/deferred_readme_set_version.rb

```diff
@@ -0,0 +1,73 @@
+require 'semantic'
+
+module Fastlane
+  module Actions
+
+    class DeferredReadmeSetVersionAction < Action
```

Comment:
Don't consider this a must fix, but I wonder if this can be made generic enough to work across our other repos? Param could be an array of strings to replace? Not sure if its worth it yet or if we just make it generic once we have another repo that wants to do this.